composer.json

```diff
@@ -17,5 +17,10 @@
         "neos": {
             "package-key": "Netlogix.Sentry"
         }
+    },
+    "config": {
+        "allow-plugins": {
+            "neos/composer-plugin": true
```

Comment:
Unfortunately that's not the composer.json used during testing. The workflows create their own composer project, so you will need to add a cli step to https://github.com/netlogix/Netlogix.Sentry/blob/master/.github/workflows/functionaltests.yml#L43

```
      - name: "[3/5] Allow neos composer plugin"
        run: composer config --no-plugins allow-plugins.neos/composer-plugin true
        working-directory: ${{ env.FLOW_DIST_FOLDER }}
```

Reply:
Oh, you are right. I've adjusted the Github Actions, now the tests are also passing.